### PR TITLE
feature: added landing page

### PR DESCRIPTION
**Pull request by omgaikwad**
feature: added landing page

### CODE REVIEW

File: css/main.css [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+  color: var(--off-white);
+  font-weight: 600;
+  font-size: 1.4rem;
+}
+.category-container {
+  display: flex;
+  /* align-items: center; */
+  justify-content: space-around;
+  flex-wrap: wrap;
+  /* height: 50rem; */
```
Comment (aryan-13) [inline css/main.css]: Commented can be removed

**Review by aryan-13 — COMMENTED** [review]
The commented code can be removed. Else everything looks good!

**Review by AnuragSahu11 — APPROVED** [review]
Looks good

File: css/main.css (hunk 1 above)
Comment (omgaikwad) [reply css/main.css]: Sure

**Pull request merged**